scripts/build-image.sh

```diff
@@ -96,7 +96,7 @@ retry '/snap/bin/lxc exec builder -- /usr/bin/nslookup github.com' 'Wait for net
 
 /snap/bin/lxc exec builder -- /usr/bin/apt-get update
 /snap/bin/lxc exec builder --env DEBIAN_FRONTEND=noninteractive -- /usr/bin/apt-get upgrade -yq
-/snap/bin/lxc exec builder --env DEBIAN_FRONTEND=noninteractive -- /usr/bin/apt-get install docker.io npm python3-pip shellcheck jq wget -yq
+/snap/bin/lxc exec builder --env DEBIAN_FRONTEND=noninteractive -- /usr/bin/apt-get install docker.io npm python3-pip shellcheck jq wget unzip gh -yq
```

Comment:
As noted in a comment, I think `gh` might not be well maintained in the apt package, could we take that over and make sure it is the latest please?

Reply:
Not quite sure what you mean by taking it over. This is a package in the ubuntu archives, and versions of packages in the archives aren't updated outside of security fixes (see https://wiki.ubuntu.com/StableReleaseUpdates). We could create a PPA that has a more up to date version, but we'd be responsible for maintaining that.

Reply:
Let's go ahead with this as is for now, and open a separate task to make this more up to date (we'll need to decide whether to create a PPA for this or a snap).